stock_picking_product_assortment/__manifest__.py

```diff
@@ -10,6 +10,6 @@
     "maintainers": ["CarlosRoca13"],
     "license": "AGPL-3",
     "installable": True,
-    "depends": ["stock", "product_assortment"],
+    "depends": ["stock", "product_assortment", "base_view_inheritance_extension"],
```

Comment:
Oooh, you add a new dependency, great ! :wink:

Reply:
Yeah!! I know that is not a good idea but this way I remove a module to be maintained

Reply:
@sergio-teruel please sort them alphabetically.

@rousseldenis is it OK for you then?

Reply:
For me, improvements even with several added modules in dependencies is always good :stuck_out_tongue_winking_eye: